docs/designers-developers/developers/block-api/block-edit-save.md

```diff
@@ -79,6 +161,7 @@ edit( { attributes, setAttributes, className, isSelected } ) {
 	);
 }
 ```
+{% end %}
 
 When using attributes that are objects or arrays it's a good idea to copy or clone the attribute prior to updating it:
 
```

Comment:
I'd love if we could ES5y the following example as well :)

Reply:
Added a clone example in ES5, so much more verbose :-)